docs/user_guide/data_prepare_cat.rst

```diff
@@ -36,6 +36,9 @@ The ``rename``, ``nodata``, ``unit_add`` and ``unit_mult`` options are set per v
       crs: EPSG/WKT
       data_type: RasterDataset/GeoDataset/GeoDataFrame/DataFrame
       driver: raster/raster_tindex/netcdf/zarr/vector/vector_table/csv/xlsx/xls
+      extent:
+        spatial: [xmin, ymin, xmax, ymax]
+        temporal: [start_time, end_time]
```

Comment:
Two quick comments for discussion: 
- It would be good to discuss the naming of these properties and also whether these should be nested under extent. In other parts of HydorMT we use e.g. `bbox` for spatial extent. 
- Also many datasets wil have a single datetime instead of start and end. Do we enforce that the temporal extent has two values?

Reply:
I'm okay with using bbox instead if that's more consistent. 
The temporal ranges are inclusive, so as written the detection code will return a tuple with the same value, (or at least it should). I could go either way on this, enforcing is much nicer for the code since it eliminates a bunch of error prone null checks, while I can imagine allowing just one value is nicer for user.

Reply:
Actually I decided to remove the stuff from the documentation since we're not adding it to the catalogs yes, however the discussion points are still good to resolve